python/sglang/srt/managers/tp_worker.py

```diff
@@ -310,11 +310,16 @@ def handle_generate_request(
         if self.model_runner.is_generation:
             req.pixel_values = recv_req.pixel_values
             if req.pixel_values is not None:
+                image_hash = (
+                    sum(recv_req.image_hash)
```

Comment:
I am confused how to make it more compatible, since `recv.req.image_hash` is a list now. We can't hash the list I think? Here we write a if/else logic to backward comaptible to previous sitatuions.